app/views/content/steps-to-become-a-teacher/_step_6_start_your_teacher_training.erb

```diff
@@ -0,0 +1,7 @@
+<p>All postgraduate training involves classroom placements in at least 2 schools, with some theoretical learning – this might be in a different location to your placements.</p>
```

Comment:
```suggestion
<p>All postgraduate training involves classroom placements in at least 2 schools. Courses also include some theoretical learning, which might be in a different location to your placements.</p>
```

@EllieNodder I wonder if it is worth splitting this into two sentences - the way it was written, I was a bit confused if the theoretical learning was in schools? (NB you can probably word better than me!)